src/app/components/MessageBanner/index.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Heading from '../Heading';
+import Image from '../Image';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface MessageBanner {
+  summaries: {
+    imageUrl?: string;
+    link: string;
+    imageAlt?: string;
+    description?: string;
+    title: string;
+    id: string;
+  }[];
+  title: string;
+  position: number;
+}
```

Comment:
Also, when we come to refactor and remove the summaries we'll probs need to make some adjustments

Reply:
I'd create a new file. Not entirely sure on the best name for it since the banner will be available across Topics and Indexes I imagine? But best to keep it separate as it doesn't really fall into any existing type file.

Reply:
Extracted the summaries as type `Summary` to the `promoData.ts` file. We don't use curations in this promo so i've not moved that definition over in this PR. Does that work?